terraform/050-pw-manager/task-definition-api.json

```diff
@@ -170,7 +170,7 @@
         "value": "${password_rule_minlength}"
       },
       {
-        "name": "PASSWORD_RULE_minScore,",
+        "name": "PASSWORD_RULE_minScore",
```

Comment:
Nice catch. Did it even work before?

Reply:
No. From what I recall, we haven't tried to change this from the default value.